Tests/Packet++Test/Tests/FtpTests.cpp

```diff
@@ -90,174 +90,179 @@ PTF_TEST_CASE(FtpParsingTests)
-		{pcpp::FtpResponseLayer::FtpStatusCode::NEED_UNAVAILABLE_RESOURCE_TO_SEC,
-		 "Need some unavailable resource to process security"},
-		{pcpp::FtpResponseLayer::FtpStatusCode::REQ_FILE_ACTION_NOT_TAKEN, "Requested file action not taken"},
-		{pcpp::FtpResponseLayer::FtpStatusCode::REQ_ACTION_ABORTED,
-		 "Requested action aborted: local error in processing"},
-		{pcpp::FtpResponseLayer::FtpStatusCode::REQ_ACTION_NOT_TAKEN,
-		 "Requested action not taken. Insufficient storage space in system"},
-		{pcpp::FtpResponseLayer::FtpStatusCode::SYNTAX_ERROR_COMMAND_UNRECOGNIZED,
-		 "Syntax error, command unrecognized"},
-		{pcpp::FtpResponseLayer::FtpStatusCode::SYNTAX_ERROR_PARAMETER_OR_ARGUMENT,
-		 "Syntax error in parameters or arguments"},
-		{pcpp::FtpResponseLayer::FtpStatusCode::COMMAND_NOT_IMPLEMENTED, "Command not implemented"},
-		{pcpp::FtpResponseLayer::FtpStatusCode::BAD_SEQUENCE_COMMANDS, "Bad sequence of commands"},
-		{pcpp::FtpResponseLayer::FtpStatusCode::COMMAND_NOT_IMPLEMENTED_FOR_PARAMETER,
-		 "Command not implemented for that parameter"},
-		{pcpp::FtpResponseLayer::FtpStatusCode::NETWORK_PROTOCOL_NOT_SUPPORTED, "Network protocol not supported"},
-		{pcpp::FtpResponseLayer::FtpStatusCode::NOT_LOGGED_IN, "Not logged in"},
-		{pcpp::FtpResponseLayer::FtpStatusCode::NEED_ACCOUNT_FOR_STORE_FILE, "Need account for storing files"},
-		{pcpp::FtpResponseLayer::FtpStatusCode::COMMAND_PROTECTION_DENIED,
-		 "Command protection level denied for policy reasons"},
-		{pcpp::FtpResponseLayer::FtpStatusCode::REQUEST_DENIED, "Request denied for policy reasons"},
-		{pcpp::FtpResponseLayer::FtpStatusCode::FAILED_SEC_CHECK, "Failed security check (hash, sequence, etc)"},
-		{pcpp::FtpResponseLayer::FtpStatusCode::REQ_PROT_LEVEL_NOT_SUPPORTED,
-		 "Requested PROT level not supported by mechanism"},
-		{pcpp::FtpResponseLayer::FtpStatusCode::COMMAND_PROTECTION_LEVEL_NOT_SUPPORTED,
-		 "Command protection level not supported by security mechanism"},
-		{pcpp::FtpResponseLayer::FtpStatusCode::FILE_UNAVAILABLE, "Requested action not taken: File unavailable"},
-		{pcpp::FtpResponseLayer::FtpStatusCode::PAGE_TYPE_UNKNOWN, "Requested action aborted: page type unknown"},
-		{pcpp::FtpResponseLayer::FtpStatusCode::EXCEED_STORAGE_ALLOCATION,
-		 "Requested file action aborted: Exceeded storage allocation"},
-		{pcpp::FtpResponseLayer::FtpStatusCode::FILENAME_NOT_ALLOWED,
-		 "Requested action not taken: File name not allowed"},
-		{pcpp::FtpResponseLayer::FtpStatusCode::INTEGRITY_PROTECTED, "Integrity protected reply"},
-		{pcpp::FtpResponseLayer::FtpStatusCode::CONFIDENTIALITY_AND_INTEGRITY_PROTECTED,
-		 "Confidentiality and integrity protected reply"},
-		{pcpp::FtpResponseLayer::FtpStatusCode::CONFIDENTIALITY_PROTECTED, "Confidentiality protected reply"}};
-
-	for (const auto &entry : possibleStatusCodes)
+		{ static_cast<pcpp::FtpResponseLayer::FtpStatusCode>(0),                          "Unknown Status Code"                             },
```

Comment:
Same here...

Reply:
This one became good.

Reply:
It still has redundant tabs, we should probably turn of clang-format...

Reply:
I'm wondering what's the purpose of a formatter that has so many bugs 😕 
I invest so much time on reviewing the code...